connectors/src/connectors/slack/index.ts

```diff
@@ -440,7 +444,9 @@ export class SlackConnectorManager extends BaseConnectorManager<SlackConfigurati
       await SlackChannel.findAll({
         where: {
           connectorId: this.connectorId,
-          slackChannelId: Object.keys(permissions),
+          slackChannelId: Object.keys(permissions).map((k) =>
+            slackChannelIdFromInternalId(k)
+          ),
```

Comment:
isn't that missing an Op.in?

Reply:
Not needed. Also if it worked, it'll work :-)